clients/vscode-hlasmplugin/README.md

```diff
@@ -21,6 +21,12 @@ HLASM Language Support is also part of [Code4z](https://marketplace.visualstudio
 
 HLASM Language Support is supported on Visual Studio Code and Github Codespaces.
 
+## Integration with Zowe Explorer
+
+We recommend that you use HLASM Language Support with Zowe Explorer. Zowe Explorer enables you to open your High Level Assembler programs stored on mainframe data sets directly in VS Code.
```

Comment:
Shouldn't we first verify that this is actually working reasonably well?

Reply:
<img width="1367" alt="Screenshot 2022-06-10 at 16 39 06" src="https://user-images.githubusercontent.com/38490125/173090024-1c60c666-b559-490e-8ff9-e23ce872a9e6.png">

Reply:
looks good to me - what do you want to test?